package.json

```diff
@@ -51,6 +51,7 @@
     "dependencies": {
         "@openzeppelin/contracts": "4.7.3",
         "@openzeppelin/contracts-upgradeable": "4.7.3",
+        "abdk-libraries-solidity": "3.1.0",
```

Comment:
the latest version is 3.2.0

Reply:
ok, updated in realated PR https://github.com/IPOR-Labs/ipor-protocol/pull/526